src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIMarsHandle.cs

```diff
@@ -475,6 +508,7 @@ private void SendAckIfNecessary()
             if (receiveHighwater - receiveHighwaterLastAck > ACK_THRESHOLD)
             {
                 SendControlPacket(SNISMUXFlags.SMUX_ACK);
+                SqlClientEventSource.Log.TryTraceEvent("SNIMarsHandle.SendAckIfNecessary | Info | MARS Session Id {0}, _asyncReceives {1}, _receiveHighwater {2}, _sendHighwater {3}, _receiveHighwaterLastAck {4} Sending acknowledgment > ACK_THRESHOLD {5}", ConnectionId, _asyncReceives, _receiveHighwater, _sendHighwater, _receiveHighwaterLastAck, ACK_THRESHOLD);
```

Comment:
"_receiveHighwaterLastAck {4} Sending acknowledgment > ACK_THRESHOLD {5}" is kind of confusing.

Reply:
It's related to MARS, of course it's confusing 😀

Reply:
I guess I mainly wanted to capture necessary properties here, when you dig down filtering, you'll understand the context :)